source/codegen/metadata/nifgen/attributes.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FGEN API metadata version 23.0.0d13
+# This file is generated from NI-FGEN API metadata version 23.0.0d36
```

Comment:
Please do another update with the latest FGEN export from this morning.

Reply:
Yeah, should be d39. We changed some parameter names.

Reply:
Also, after you make the update to the imported metadata/proto make sure to build locally to get updated files in the generated folder. Otherwise, the PR build will fail.

Reply:
It would be nice to mention the version you ended up using somewhere in the PR description.